portfolio/src/main/webapp/hobbies.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>My Hobbies</title>
+        <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
+        <link rel="stylesheet" href="style.css">
+        <script src="script.js"></script>
+    </head>
+    <body>
+      <div id="barTop">
+        <nav class="navbar navbar-expand-lg navbar-dark" style="background-color: #004047">
+          <a class="navbar-brand" href="#">Andrés's Portfolio</a>
```

Comment:
Do you want this to link to this current hobbies page or your home page? Typically the navbar logo/brand would go to the homepage, so index.html in this case. (But if you'd prefer it to point to the hobbies page, just let me know!)

Reply:
Done! It was a mistake, the brand logo now points to homepage!